.travis.yml

```diff
@@ -11,7 +11,8 @@ branches:
 env:
   global:
   - AWS_DEFAULT_REGION: ap-southeast-2
+  # TRAVIS_TOKEN
   - secure: Y7BAj6oYGbLU5jNswZrWLH5m5tPuitlsbKgW6luZ2FKBKTnkl77p0hyb+KHbg3Gzg7meQ971SKhnd5UgGecaz6Ych40GW16/gYPEUDxMIs12/bODYTlsdE3wgHk2IzjAa4SShO20kJgeFi4jhUzxSNo6oYV0o+EHMmtT2oHaEMA=
-  - secure: gMOlEqMgtwKnIpe9A9j2r/vp80iNl8X9ZPshB1h+6Ensu1QA1Rxtrlb4z8Sknss5bJcpNeLkBd/xPHABIgR84JUuIl8cWFn/zNJwTNdoHH70NLziBuLoN4efs/yk6+wh3NsJJuFuOMKvjNUTg6cCNR1CVuoXLm/1Y5s4WV09Zv0=
+  - AWS_ACCESS_KEY_ID: AKIAJBSOFDJSDGWCKQ3Q
+  # AWS_SECRET_ACCESS_KEY, /geodesy-operations/iam-user/geodesy-operations-pipelines-dev/access-key/AKIAJBSOFDJSDGWCKQ3Q/secret-access-key
```

Comment:
I can't find the parameter you are referencing in SSM.

Reply:
Currently, dev and test passwords are in gnss-common-nonprod and prod passwords are in gnss-common-prod.

Reply:
Here is what's in nonprod now:
```
nix-shell geodesy/gnss-common$ aws ssm describe-parameters | grep -i name
            "Name": "/dockerhub/user/geodesyarchive/password",
            "Name": "/geodesy-operations/iam-user/bitbucket-pipelines/access-key/AKIA2AV2YTL6QKCM4I5F/secret-access-key",
            "Name": "/github/user/ga-gnss-ci/password",
            "Name": "/gnss-common-nonprod/iam-user/bitbucket-pipelines/access-key/AKIAQKXZ645FRVJEU3OZ/secret-access-key",
            "Name": "/gnss-common-prod/iam-user/bitbucket-pipelines/access-key/AKIAU3Z3JZ7MJI2PL53G/secret-access-key",
            "Name": "/gnss-data-nonprod/iam-user/bitbucket-pipelines/access-key/AKIAQKXZ645F2LNIHL7E/secret-access-key",
            "Name": "/gnss-metadata-nonprod/iam-user/travis-ci/access-key/AKIA2H4MYP4OFLXHC77W/secret-access-key",
            "Name": "/gnss-rds-dev/user/master/password",
            "Name": "/gnss-rds-test/user/master/password",
```

Reply:
Do get the value out you say
```
aws ssm get-parameter --name /dockerhub/user/geodesyarchive/password --with-descryption
```

Reply:
To put something in you say:
```
aws ssm put-parameter --type SecureString --name /dockerhub/user/geodesyarchive/password --value 'akdhkg'
```

Reply:
In this case, the user you want is probably /geodesy-operations/iam-user/bitbucket-pipelines/access-key/AKIA2AV2YTL6QKCM4I5F/secret-access-key.

Reply:
I just checked: the user "geodesy-operations-pipelines-dev" I used is in geodesy-operations and was created nearly 3 years ago. User "bitbucket-pipelines" was created 16 days ago.
So, I can change to use IAM user bitbucket-pipelines's credentials for repos.

Reply:
yes, then you want need to enter a new password into SSM, it's already there, you can just use the command above to get the value of the password out of SSM

Reply:
You sent a PR yesterday, would that need adjusting also?

Reply:
Yes, I'll do the same change to GeoesyML repo.

Reply:
I'll delete the IAM user "geodesy-operations-pipelines-dev" from geodesy-operations once we switch to use "bitbucket-pipelines" credentials for all repos.

Reply:
What is currently using user geodesy-operations-pipelines-dev? What will brake when you delete the user?

Reply:
AWS says that the user was last accessed today. Maybe we can leave the user for a bit, to make sure that nothing is using it.

Reply:
Ok, I'll leave the user "geodesy-operations-pipelines-dev" for a while.
The "`AWS_ACCESS_KEY_ID: AKIAJBSOFDJSDGWCKQ3Q`" comes from user "geodesy-operations-pipelines-dev", and is currently used by xml-schemer, GeodesyML, GeodesyML-Java_Bindings, IGSSITELOG-Java-Bindings, and GeodesyMLConverter. It was also used by Geodesy-Web-Services before you changed to use the credentials of "bitbucket-pipelines" yesterday.